src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Program.Main.cs

```diff
@@ -51,6 +51,7 @@ public static void Main(string[] args)
 
         #endif
         app.UseStaticFiles();
+        app.UseAntiforgery();
```

Comment:
Do we need this?

Blazor already does its own antiforgery check.

Reply:
We have a check in endpoint routing to validate that the anti-forgery middleware was properly configured for non-Blazor scenarios (see [this](https://github.com/dotnet/aspnetcore/blob/2335aa56f519790c4b4b53b955b26f29f7030d36/src/Http/Routing/src/EndpointMiddleware.cs#L53-L57)). We could exempt Blazor from this check but I'd rather have the behavior consistent across implementations.